docs/nerfology/methods/nerf2gs2nerf.md

```diff
@@ -0,0 +1,86 @@
+# From NeRFs to Gaussian Splats, and Back
+
+This is the implementation of [From NeRFs to Gaussian Splats, and Back](https://arxiv.org/abs/2405.09717); An efficient procedure to convert back and forth between NeRF and GS, and thereby get the best of both approaches. New dataset in the paper can be downloaded from [this google drive link](https://drive.google.com/drive/folders/1xvbONL4EVgHxaHMsV101455l_jNgyaUM?usp=sharing).
+
+<video id="teaser" muted autoplay playsinline loop controls width="100%">
```

Comment:
@Siming-He, really interesting project. I don't fully understand the demo video though, can you briefly describe what the left and right columns depict, mainly the right column. It switches from nerf-sh to nerf-gs (what exactly does that mean)? SH is the nerf variant and then you convert it to the Nerf-GS variant or do you train again to generate the Nerf-GS variant?

Reply:
Hi, I updated my branch with the latest main. For the video, NeRF-SH is the nerf variant that predicts spherical harmonic coefficients. I directly convert it to NeRF-GS with conversion time 7s (which is Gaussian splats extracted from NeRF-GS). Then I finetune NeRF-GS on the training data (for 3s, 33s, and 153s) to get better NeRF-GS. Thank you for the question! I'll add the explanations into the video.